astroneer/egg-astroneer-dedicated-server.json

```diff
@@ -30,7 +30,7 @@
     "variables": [
         {
             "name": "Public IP",
-            "description": "Public IP to connect to the server.",
+            "description": "Public IP to connect to the server (Don't include the port).",
```

Comment:
I hate making descriptions more verbose and less accurate. I don't want to perpetuate the misdefinition of what an IP address is.

Instead, lets try just applying validation to prevent the mistakes this is trying to counter. `required|ip` should work.